core/src/main/java/com/datastax/oss/driver/internal/core/os/Native.java

```diff
@@ -25,8 +25,24 @@
 
   private static class LibcLoader {
 
+    /* These values come from Graal's imageinfo API which aims to offer the ability to detect
+    * when we're in the Graal build/run time via system props.  The maintainers of Graal have
+    * agreed that this API will not change over time.  We reference these props as literals
+    * to avoid introducing a dependency on Graal code for non-Graal users here. */
```

Comment:
```suggestion
     * when we're in the Graal build/run time via system props.  The maintainers of Graal have
     * agreed that this API will not change over time.  We reference these props as literals
     * to avoid introducing a dependency on Graal code for non-Graal users here. */
```
To make the formatter happy 🙂

Reply:
Like I said earlier, man... I'm all about making the tools happy. ;)